src/serializer/functions.js

```diff
@@ -57,26 +61,40 @@ export class Functions {
   writeEffects: WriteEffects;
   _noopFunction: void | ECMAScriptSourceFunctionValue;
 
+  __unwrapAbstract(value: AbstractValue): Value {
```

Comment:
I don't know how the double-underscore trend for non-private function function names got started. Please only use single-underscores.